topics/psr-15/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: "PSR-15: HTTP Server Request Handlers"
+short_description: PSR-7 is a set of common interfaces representing HTTP server request handlers, and HTTP server middleware components.
+topic: psr-15
+related: psr, psr-7
+url: https://www.php-fig.org/psr/psr-15/
+created_by: Woody Gilk
+---
+common interfaces for HTTP server request handlers ("request handlers") and HTTP server middleware components ("middleware") that use HTTP messages as described by PSR-7 or subsequent replacement PSRs.
```

Comment:
This sentence seems incomplete?

Reply:
I prependend the start of the sentence. Does that look better now?